src/components/Home/styles.css

```diff
@@ -0,0 +1,51 @@
+
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css");
+
+.btn-primary, btn-primary:hover, btn-primary:active, btn-primary:visited {
+    background-color: blue !important; 
+    color: white !important;
+    border: none !important;
+}
+
+.btn-secondary, btn-secondary:hover, btn-secondary:active, btn-secondary:visited {
+    background-color: #EEF1EF !important;
+    color: black !important;
+    border: none !important;
+}
+
+.top-buffer {
+    margin-top: 140px !important;
+}
+
+.mid-buffer {
+    margin-top: 70px !important;
+}
+
+  
```

Comment:
This is overriding some styling in the Account pages & can probs be removed. Tested locally and works well!

Reply:
Yeah, that would probably be a good idea. When I first did this file I just copy/pasted it from Account. Then later I added an import of the Account stylesheet into the Home.js file so changes there would also apply here. Forgot about this.